pkg/operator/sync_v310_00.go

```diff
@@ -23,6 +23,100 @@ import (
 
 // most of the time the sync method will be good for a large span of minor versions
 func sync_v310_00_to_00(c ServiceCertSignerOperator, operatorConfig *scsv1alpha1.ServiceCertSignerOperatorConfig, previousAvailability *operatorsv1alpha1.VersionAvailablity) (operatorsv1alpha1.VersionAvailablity, []error) {
+	signingVersionAvailability, signingErrors := sync_v310_00_to_00_SigningController(c, operatorConfig, previousAvailability)
+	apiServiceInjectorVersionAvailability, apiServiceInjectorErrors := sync_v310_00_to_00_APIServiceController(c, operatorConfig, previousAvailability)
+
+	allErrors := []error{}
+	allErrors = append(allErrors, signingErrors...)
+	allErrors = append(allErrors, apiServiceInjectorErrors...)
```

Comment:
allErrors := append(signingErrors, apiServiceInjectorErrors...)

Reply:
david's variant is more uniform.

Reply:
gotcha, I'm fine with that